ner/README.md

```diff
@@ -172,6 +172,20 @@ If set to "true", this may potentially add some robustness if you add
 sophisticated features that depend on deeper levels of NLP processing
 (currently, very few are used/required). 
 
+
+### How to use the new parameter 'useFE'
+useFE is used to decide whether we want to use Frustratingly Easy Domain Adaptation or not. The default value is false. If useFE is true, then the structure of the training data folder must follow the structure like
```

Comment:
I think you have to give more details. Like you mention how things should be different for training. How about a user who doesn't care about training; does it make any different for such user?